spec/controllers/games_controller_spec.rb

```diff
@@ -91,15 +91,5 @@
       expect(game.black_player_id).to eq(user2.id)
       expect(player2.username).to eq(user2.username)
     end
-
-    it "will not allow two players with the same ID" do
```

Comment:
Are we removing this as a requirement? Or are we testing this elsewhere?

Reply:
Yup! Before we didn't have a great way to handle it, but I tweaked the controller a little. We should let users access their own games as long as they are active. They won't be added as the other player at all. They'll just load right back into their game.

Reply:
Awesome! Sounds good!